training/model_evaluation/environments/context/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20220516.v1
+
+ENV AZUREML_CONDA_ENVIRONMENT_PATH /azureml-envs/model-eval-env
+# Create conda environment
+RUN conda create -p $AZUREML_CONDA_ENVIRONMENT_PATH \
+    python=3.8
+
+# Prepend path to AzureML conda environment
+ENV PATH $AZUREML_CONDA_ENVIRONMENT_PATH/bin:$PATH
+
+
+# Install conda dependencies
+RUN conda install scikit-learn==0.24.1
+# Install pip dependencies
+RUN pip install 'matplotlib>=3.3,<3.4' \
```

Comment:
Please include these in a requirements.txt file, and use template tags to get the latest versions of the SDK or other packages. See examples here: https://github.com/Azure/azureml-assets/tree/main/training/general/src/environments